### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by origamiserviceuser**
:robot: I have created a release *beep* *boop*
---

<details><summary>o-header: 14.0.1</summary>

### Bug Fixes

* (o-header) hide sticky header container when sticky header disappears ([027b716](https://github.com/Financial-Times/origami/commit/027b7167f3a49a2b637f33e25d302da44320d039))
* (o-header) increase the value of `o-header` container to `10` ([3d74d56](https://github.com/Financial-Times/origami/commit/3d74d56db277e44ffc06c00ea081611ca6de3f68))
* (o-header) remove `padding` from the form label ([bd1f0af](https://github.com/Financial-Times/origami/commit/bd1f0af0534b9c68fb49c8730d3fc2a8614f3733))
</details>

<details><summary>o3-form: 0.3.0</summary>

### Features

* add text-input component ([#1786](https://github.com/Financial-Times/origami/issues/1786)) ([fdbadfe](https://github.com/Financial-Times/origami/commit/fdbadfea1f65ed24b554c042f1cb899bc6301e5b))
</details>

<details><summary>o3-foundation: 1.6.0</summary>

### Features

* add text-input component ([#1786](https://github.com/Financial-Times/origami/issues/1786)) ([fdbadfe](https://github.com/Financial-Times/origami/commit/fdbadfea1f65ed24b554c042f1cb899bc6301e5b))
* introduce `o3-grid-column-width` CSS Custom Property ([#1776](https://github.com/Financial-Times/origami/issues/1776)) ([9ff37f4](https://github.com/Financial-Times/origami/commit/9ff37f46d969161f117aefb13ea44d58becfbb97))

### Bug Fixes

* Remove prefers-reduced-motion from normalise.css ([#1801](https://github.com/Financial-Times/origami/issues/1801)) ([d5d39f1](https://github.com/Financial-Times/origami/commit/d5d39f1a88eb0e02d9d7eec895ee9260092b680a))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: components/o3-foundation/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.6.0](https://github.com/Financial-Times/origami/compare/o3-foundation-v1.5.0...o3-foundation-v1.6.0) (2024-08-27)
+
+
+### Features
+
+* add text-input component ([#1786](https://github.com/Financial-Times/origami/issues/1786)) ([fdbadfe](https://github.com/Financial-Times/origami/commit/fdbadfea1f65ed24b554c042f1cb899bc6301e5b))
```
Comment (frshwtr) [inline components/o3-foundation/CHANGELOG.md:8]: The message is misleading here and should be updated. This change description makes it sound like text-input has been added to `o3-foundation` which is not the case. Only use case tokens that support this component have been added to foundation.

**Review by frshwtr — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-header-v14.0.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o3-form-v0.3.0 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o3-foundation-v1.6.0 :sunflower:

**Pull request merged**